nilearn/decoding/decoder.py

```diff
@@ -876,3 +921,301 @@ def __init__(self, estimator='svr', mask=None, cv=10, param_grid=None,
             mask_strategy=mask_strategy, memory=memory,
             is_classification=False, memory_level=memory_level,
             verbose=verbose, n_jobs=n_jobs)
+
+
+class fREMRegressor(_BaseDecoder):
```

Comment:
Just for the naming convention I thought class should be CapName? (e.g. in https://www.python.org/dev/peps/pep-0008/#id41)

Reply:
I wondered about it. So the method is called fREM but the convention would mean using Frem (exact opposite). So I'm not sure which one is more confusing for users.

@emdupre ?

Reply:
While Classes are supposed to LikeThis, for symbolism with a specific meaning & serving a specific purpose, it is fine to change it a bit.
I prefer `fREM` over `Frem`.